### DISCUSSION TITLE
Unable to join the Slack group for Feathr.

### DISCUSSION THREAD

**Discussion by masoudshab** [feathr-ai/feathr#290] [Q&A] [answered]
We will be using Microsoft LinkedIn Feathr Feature Store for our enterprise use case. Have couple of critical questions. Would be great if I can join the slack group for technical issues.

![image](https://user-images.githubusercontent.com/36493321/169601262-aea8897f-e13f-446d-b685-3c145c1e1b27.png)

**Comment by xiaoyongzhu** [comment]
fixed per https://github.com/linkedin/feathr/issues/291

**Reply by garimss** [reply]
I can't use either. Can you provide the latest link please? Thanks

**Comment by hangfei** [comment, marked as answer]
Use https://join.slack.com/t/feathrai/shared_invite/zt-1ffva5u6v-voq0Us7bbKAw873cEzHOSg.

**Comment by SaumyaBhatt106** [comment]
Hey, the above link still isn't working
